Dalamud/Game/ClientState/ClientState.cs

```diff
@@ -178,12 +177,102 @@ private void FrameworkOnOnUpdateEvent(Framework framework1)
 
             if (this.IsPvP)
             {
-                this.EnterPvP?.InvokeSafely();
+                this.EnterPvP?.Invoke();
```

Comment:
Can you please clarify why you're switching these to `Invoke` over `InvokeSafely`?

Reply:
While adding interfaces, I have been declaring events in the public interface as non-nullable, and changing the internal events to nullable. The IDE doesn't complain about this and provides nice intellisense both internally and externally.

InvokeSafely was used on the non-nullable objects to check for null and then invoke, this is not necessary anymore as the internal events are properly nullables now. Rather than using a dalamud wrapper for invoke, I felt it was best to use the language features directly.

Reply:
I will note that [`InvokeSafely`](https://github.com/goatcorp/Dalamud/blob/130a57f850660992c12ca426cc0510f0b7f253f8/Dalamud/Utility/EventHandlerExtensions.cs#L58) also wraps all actions in a [generic try-catch](https://github.com/goatcorp/Dalamud/blob/130a57f850660992c12ca426cc0510f0b7f253f8/Dalamud/Utility/EventHandlerExtensions.cs#L86) to resolve any uncaught exceptions. Is this no longer needed/desired for our API surface?

Reply:
Good point, do we want to InvokeSafely in the ScopedService instead? This way exceptions would be caught per-plugin, instead of bubbling up to the core service?

Reply:
Nah, keep them in the core service. If we have an issue or an uncaught exception in Dalamud itself somehow, it'd be good to handle those in the same way.

I also don't see a particularly compelling reason to catch exceptions per-plugin, as there's not much that we would be able to do at the plugin-scope level (and the core stacktrace will already point at the faulting plugin).